packages/web/src/components/artist/ArtistSupporting.tsx

```diff
@@ -55,16 +57,29 @@ export const ArtistSupporting = (props: ArtistSupportingProps) => {
     return rankedSupportingList
       .sort((s1, s2) => s1.rank - s2.rank)
       .map(s => usersMap[s.receiver_id])
+      .filter(Boolean)
   })
 
+  /**
+   * It's possible that we don't have the data for which artists
+   * this artist is supporting. Thus, we fetch in this case.
+   */
+  useEffect(() => {
```

Comment:
wondering if we should do this for users as soon as we know it's possible for them to have an artist popover tile (rather than wait until artist card is shown and have the supporting section eventually show up in the card)
i.e. for all users in the user list modals, where hovering on the user would show their artist card.

Reply:
good callout - we would potentially fire a *lot* of requests for supporters with this approach (imagine quickly scrolling through trending...), so I think we should opt to stick with the approach you have in this PR